src/main/java/cs361/battleships/models/Submarine.java

```diff
@@ -10,7 +10,17 @@
 	public Submarine(String kind) {
 		this.health = 2;
 		this.size = 5;
-		this.kind = kind;
+		this.submerged=false;
+		if(kind.equals("SUBMARINE")) this.submerged = true;
+		else this.submerged = false;
```

Comment:
this is a good addition to the class. it makes way more sense to determine a true of false with a boolean instead of a string.